pycytominer/cyto_utils/load.py

```diff
@@ -1,8 +1,48 @@
 import csv
 import gzip
+from pathlib import Path
```

Comment:
is `pathlib` a large package? If not, then I think it's best to `import pathlib` and use `pathlib.Path` instead of just `Path` (to increase readability)